templates/docs/examples/patterns/tiered-list/with-default-width-image.html

```diff
@@ -0,0 +1,98 @@
+{% extends "_layouts/examples.html" %}
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
+{% block title %}Tiered list / 50/50 on desktop with description{% endblock %}
```

Comment:
I think this title is still copy-pasted from another example, could you update it so it stands out in the examples list please?